website/source/guides/operations/vault-ha-consul.html.md

```diff
@@ -25,6 +25,15 @@ redirect requests to the _active_ node.
 
 ![Reference Architecture](/assets/images/vault-ha-consul-3.png)
 
+> **NOTE:** As of version **0.11**, those standby nodes can handle read-only
```

Comment:
You may want to say "most read-only requests" since there are still some that require going to the primary.

Reply:
Updated it!  Thx :)